client/plots/disco/launch.adhoc.ts

```diff
@@ -1,8 +1,6 @@
-/*
-TODO convert to ts
-*/
+import { Genome } from '#shared/types/index'
 
-export async function launch(arg, genomeObj, holder) {
+export async function launch(arg: any, genomeObj: Genome, holder: Element) {
```

Comment:
rather than using "any" for `arg`, please declare the possible attributes based on my comments before `getMlst()`